config/swagger.yaml

```diff
@@ -402,6 +402,434 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security:
+  /accounts/{pubkey}:
```

Comment:
Re all of these paths that expose state trees: do you have any requirements for querying at specific hash or height? (Just a reminder, no issue with this path.)

Reply:
There is no requirement for querying at height/hash